src/libtriton/ast/z3/tritonToZ3Ast.cpp

```diff
@@ -31,7 +31,11 @@ namespace triton {
         throw triton::exceptions::Exception("TritonToZ3Ast::getUintValue(): The ast is not a numerical value.");
 
       #if defined(__x86_64__) || defined(_M_X64)
-      Z3_get_numeral_uint64(this->context, expr, &result);
+          #if defined(__uint64)
+              Z3_get_numeral_uint64(this->context, expr, (__uint64*) &result);
+          #else
+              Z3_get_numeral_uint64(this->context, expr, (uint64_t*) &result);
+          #endif
```

Comment:
This is fixed into the `dev-v0.6`.

Reply:
I see. Thank you. I remove this commit.
https://github.com/JonathanSalwan/Triton/issues/679